Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.11-slim
```

Comment:
good choice on slim. Some extra reading about [it vs alpine](https://www.reddit.com/r/docker/comments/g5hb93/alpine_vs_pythonslim_for_deploying_python_data/) if you're curious.

Reply:
Yeah, the problems building wheels is why I went with slim.